src/codegen/c.zig

```diff
@@ -392,6 +392,24 @@ pub const DeclGen = struct {
                     try dg.renderType(writer, ty);
                     try writer.print(")0x{x}u)", .{val.toUnsignedInt()});
                 },
+                .field_ptr => {
+                    const field_ptr = val.castTag(.field_ptr).?.data;
+                    const index = field_ptr.field_index;
+                    const container_ptr = field_ptr.container_ptr;
+                    const decl = switch (container_ptr.tag()) {
+                        .decl_ref => container_ptr.castTag(.decl_ref).?.data,
```

Comment:
I'm not quite sure what values I need to handle here. I've seen in other places in this file it handles a subset of tags that you might expect (eg decl_ref but not decl_ref_mut)